winit-core/src/keyboard.rs

```diff
@@ -1687,9 +1687,10 @@ pub enum KeyLocation {
 }
 
 bitflags! {
-    /// Represents the current state of the keyboard modifiers
+    /// Represents the current logical state of the keyboard modifiers
     ///
-    /// Each flag represents a modifier and is set if this modifier is active.
+    /// Each flag represents a modifier and is set if this modifier is active
+    /// (but the modifier key can be physically released as in the case of sticky modifiers).
```

Comment:
Nit: Written a bit clearer IMO.

```suggestion
    /// Each flag represents a modifier and is set if this modifier is active.
    ///
    /// Note that the modifier key can be physically released with the modifier
    /// still being marked as active, as in the case of sticky modifiers.
```